spec/controllers/your_details_controller_spec.rb

```diff
@@ -100,6 +102,44 @@ module Wpcc
       end
     end
 
+    describe 'sessions' do
+      after { Timecop.return }
+
+      it "'wpcc_expires_at' key should always be updated" do
+        session[:wpcc_expires_at] = 5.minutes.ago
+        get :new, nil, session
+
+        expect(controller.session[:wpcc_expires_at]).to be > 5.minutes.ago
+      end
+
+      context 'session has expired' do
+        it 'should reset the wpcc specific session data to nil' do
+          session[:wpcc_expires_at] = Time.current
+          Timecop.travel(Time.current + 30.minutes)
+          get :new, nil, session
+
+          expect(session[:age]).to eq nil
```

Comment:
Humm, interesting. Because he is using the session from the let. Perhaps the let(:session) should be renamed to something else is more important than controller.session.